### ISSUE TITLE
Neural net organization and parameters

### ISSUE THREAD

**Issue by chiggum** [spragunr/deep_q_rl#10]
Hi spragunr,

I am really interested in the deep reinforcement learning and m trying to build a similar agent for breakout on my own (though it's getting really tough for me to get my agent learn better). I have gone through your code once and have fairly understood it. Can you please spare some time of yours for some of my questions. Here are they:

1. What is the organization of layers in your neural net? I mean, from my understanding, you have a convolution layer, then another convolution layer, then a fully connected layer with relu activation and finally a fully connected layer with linear activation units. Specifically, I cannot see a relu activation after first and second conv. layers. Is it so? or am I missing something. Also, if you aren't using any activation after conv. layer 1 and 2 then can you please reason that for me.

2. Which is better (for training agent for breakout) : rmsprop or rmsprop with nesterev momentum? OR both are fairly good and give almost similar results?

3. Is your learning rate : 0.0002 fixed or decays after some fixed number of steps. If it decays then can you please tell me the number of steps after which it decays?

4. What is decay in your code? I can see that you are using a default value of 0.9 for decay but I cannot see where it is used.

5. According to my understanding, the filter units in your neural net's layers are initialized with a normal distribution with mean 0 and standard deviation 0.01 for weights and fixed value of 0.1 for bias. Is it so?

6. Finally, following are some of the parameters for breakout that you are using:
rho = 0.9
discount = 0.9
epsilon = 10e-6
Am i correct?

Sorry for so many questions. Looking at some of the work by you and others have really confused me in the context of parameter setting and neural net organization. It would be really helpful and nice of you to answer my questions.
Thanks for your time.
Regards.

**Comment by spragunr** [comment]
Hi @chiggum.  I would encourage you to post general questions like this to the discussion board.  Some of your questions are answered in my response to issue #9.  If you want to post any remaining questions to the discussion board, I would be happy to answer them there.
